manifests/config.pp

```diff
@@ -47,6 +47,14 @@
     ensure => absent,
   }
 
+  file { $foreman::client_ssl_cert:
+    owner => $foreman::user,
```

Comment:
I'm OK with a policy of `640` for keys root:<service_user>. Ensuring that this gets the group feels correct to me since this services needs to ensure it can read the file.